mobile/src/androidTest/java/ch/epfl/sdp/mobile/test/state/StatefulHomeTest.kt

```diff
@@ -240,10 +241,67 @@ class StatefulHomeTest {
 
     rule.onNodeWithText(strings.sectionPlay).performClick()
     rule.onNodeWithText(strings.newGame).performClick()
+    rule.onNodeWithText(strings.prepareGamePlayOnline).performClick()
 
     rule.onNodeWithText("user2").assertExists()
     rule.onNodeWithText(strings.prepareGameCancel).performClick()
     rule.onNodeWithText("user2").assertDoesNotExist()
-    rule.onNodeWithText(strings.newGame).assertExists()
+    rule.onNodeWithText(strings.prepareGamePlayOnline).assertExists()
+  }
+
+  @Test
+  fun given_StatefulHome_When_creatingOnlineGameFromUI_opensGameScreenWithCorrectOpponent() =
```

Comment:
Missing `then` keyword.

Reply:
Keywords start with a lowercase and between keyword we use camel case